backend/src/storage.py

```
@@ -0,0 +1,169 @@
+import boto3
+from botocore.client import Config as BotoConfig
+from botocore.exceptions import ClientError
+
+from src.settings import config
+
+logger = logging.getLogger(__name__)
+
+
+class S3Storage:
+    """Handle S3 storage operations for images."""
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)